src/libraries/System.Net.Http/tests/FunctionalTests/ResponseStreamConformanceTests.cs

```diff
@@ -35,6 +35,26 @@ protected override async Task<StreamPair> CreateConnectedStreamsAsync()
         }
     }
 
+    public sealed class Http1ConnectResponseStreamConformanceTests : ResponseConnectedStreamConformanceTests
+    {
+        protected override string GetResponseHeaders() => "HTTP/1.1 200 OK\r\n\r\n";
+
+        protected override HttpRequestMessage GetHttpRequestMessage()
+        {
+            var request = new HttpRequestMessage(new HttpMethod("CONNECT"), $"http://doesntmatter:12345/");
+            request.Headers.Host = "doesntmatter:12345";
+            return request;
+        }
+
+        protected override async Task<StreamPair> CreateConnectedStreamsAsync()
+        {
+            StreamPair pair = await base.CreateConnectedStreamsAsync();
+            Assert.True(pair.Stream2.CanWrite);
+            Assert.True(pair.Stream2.CanRead);
+            return pair;
+        }
+    }
```

Comment:
Is this testing a different stream than Http1UpgradeResponseStreamConformanceTests is testing?

Reply:
It's the same stream, but a different way of constructing it.

Reply:
From a "we know the implementation" perspective, this doesn't seem like it's actually testing anything further: 
```C#
                else if (ReferenceEquals(normalizedMethod, HttpMethod.Connect) && response.StatusCode == HttpStatusCode.OK)
                {
                    responseStream = new RawConnectionStream(this);
                    _connectionClose = true; 
                }
                else if (response.StatusCode == HttpStatusCode.SwitchingProtocols)
                {
                    responseStream = new RawConnectionStream(this);
                }
```
but if you really think it's worth adding additional stream tests for, ok.

Reply:
Yeah, I think you're right here. I'll just close this.